004/test_zombie_sim.py

```diff
@@ -0,0 +1,45 @@
+from zombie_outbreak_sim import City, simulate_zombie_outbreak
+
+
+def test_simulation():
+    """
+    Test that the simulate_zombie_outbreak function behaves as expected.
+    """
+
+    initial_grid = [
+        ["H", "H", "H", "Z"],
+        ["H", "H", "E", "H"],
+        ["H", "H", "H", "H"],
+        ["H", "H", "H", "H"],
```

Comment:
@harrymunro added some empties, as per the challenge description. Just adds a tiny bit more complexity to creating the solution, as participants will need to advance the simulation differently depending on the contents of the cell